apps/volunteers/urls.py

```diff
@@ -4,7 +4,7 @@
 
 app_name = 'volunteers'
 urlpatterns = [
-    path('', views.index, name='index'),
```

Comment:
Ah! I think `index` would have been a better choice. Sorry for the trouble but can you please change it back to `index` and use `index` in place of `volunteers`?

Reply:
Yeah, done.